API.md

```diff
@@ -1183,58 +1244,6 @@ Use this route to retrieve a FHIR Bundle containing the monitoree, all their lab
 
 The API supports creating new monitorees.
 
-#### Extensions
-
-<a name="create-ext"/>
```

Comment:
GitHub won't let me comment on the truly relevant line, but it looks like the entry for this link is still in the table of contents.

Reply:
Oops good call. Removed https://github.com/SaraAlert/SaraAlert/pull/521/commits/023530c951ffd85bc5212f02a79143facb786516